### DISCUSSION TITLE
custom mod multiplier in multiplayer or playlist

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#20531] [Ideas]
An example of this requirement is EZ and NF

Most matches set EZ and NF multiplier to a higher multiplier like 1.3x(for EZ), 1.0x(for NF).

It can be applied in mod settings and only for this room

**Comment by peppy** [comment]
I don't foresee this happening. It's too many layers of complexity. We could add a setting to turn off multipliers though, would that help?

**Reply by cdwcgt** [reply]
This only helps in NF mod like cases
But it would be nice to do this first, better than nothing

**Reply by Doksha** [reply]
just having a mod to set the total mod multiplier for local an multi scores. would not show up on the global leaderboards.
